opm/simulators/flow/BlackoilModelEbos.hpp

```diff
@@ -594,21 +594,61 @@ namespace Opm {
 
             auto& ebosJac = ebosSimulator_.model().linearizer().jacobian().istlMatrix();
             auto& ebosResid = ebosSimulator_.model().linearizer().residual();
+            auto& ebosSolver = ebosSimulator_.model().newtonMethod().linearSolver();
 
-            // set initial guess
-            x = 0.0;
+            const int numSolvers = ebosSolver.numAvailableSolvers();
+            if ((numSolvers > 1) && (ebosSolver.getSolveCount() % 100 == 0)) {
```

Comment:
Do we envisage making this constant `100` configurable at some point?

Reply:
Eventually, but at that point we should probably look at more advanced strategies as well, and think of a good name for it. I suggest leaving it for now.